Src/FluentAssertions/Primitives/ObjectAssertions.cs

```diff
@@ -220,6 +220,15 @@ public AndConstraint<TAssertions> NotBe(TSubject unexpected, string because = ""
         return new AndConstraint<TAssertions>((TAssertions)this);
     }
 
+#pragma warning disable CA1065 // "Exceptions should not be raised in this type of method..."; this is intentional to ensure that FluentAssertions' users get a suggestion of what to use instead.
```

Comment:
The linter was mad about this, so disabled that warning since this is intended.

Reply:
The other classes overriding `Equals` has this in the top, e.g `ReferenceTypeAssertions.cs:9`
```
#pragma warning disable CS0659 // Ignore not overriding Object.GetHashCode()
#pragma warning disable CA1065 // Ignore throwing NotSupportedException from Equals
```

then you can also remove the override of `GetHashCode`

Reply:
ok very good will do that.

Reply:
This is addressed.